Sass edits apr 7 2020

jhogue opened the pull request:
Various things. Notes in each commit message about what I am adding or editing and sometimes a Why.

jhogue committed:
Margin too large

jhogue committed:
Add some details for images

jhogue committed:
Add another breakpoint

Reduce the smaller max-width

jhogue committed:
Fiddle with heading sizes

jhogue committed:
Correct the font size declaration

16px will override an individual’s browser setting if they would rather have larger text as their default

jhogue committed:
Smaller mobile breakpoint, more space around hr

jhogue committed:
Add “Sold” snipe

jhogue committed:
Hide the homepage callout for now

Bummer. My design and Kay’s code were useless

kangabell commented on sass/media/_images.scss:
```diff
@@ -0,0 +1,33 @@
+	margin: 0 -1rem;
+
+  @media ($xlarge-up) {
+		margin: 0 -2.5rem;
```
indent

jhogue replied:
FUUUUUCK

jhogue committed:
Convert spaces to tabs

kangabell commented on sass/site/primary/_posts-and-pages.scss:
```diff
@@ -149,6 +149,37 @@
+  	overflow: hidden;
+  	
+  	&::after {
+    	background-color: #b5161c;
```
indent

kangabell commented on sass/site/primary/_posts-and-pages.scss:
```diff
@@ -149,6 +149,37 @@
+    	background-color: #b5161c;
+    	color: #fff;
+    	font-size: 1.125rem;
+    	font-weight: bolder;
```
this is not a weight used on this site

kangabell commented on sass/site/primary/_posts-and-pages.scss:
```diff
@@ -149,6 +149,37 @@
+  	
+  	&::after {
+    	background-color: #b5161c;
+    	color: #fff;
```
use a variable

kangabell commented on sass/site/primary/_posts-and-pages.scss:
```diff
@@ -149,6 +149,37 @@
+    	font-size: 1.125rem;
+    	font-weight: bolder;
+    	content: 'SOLD';
+    	letter-spacing: 1px;
```
should be em's not px

kangabell commented on sass/site/primary/_posts-and-pages.scss:
```diff
@@ -149,6 +149,37 @@
+  }
+}
+
+.tag-sold .entry-content .ngg-gallery-thumbnail a::after {
```
this is very high specificity. I'm wondering if it's really necessary, but it's also fine.

jhogue replied:
I know, i'm not crazy about it but I didn't also want to go through the work of creating a specific template or PHP handler for the sold tag

kangabell replied:
hmm yeah I think PHP would be more appropriate than a pseudo-element, but I see why you don't wanna bother.
I'd think `.tag-sold .ngg-gallery-thumbnail a::after` would work to simplify the selector, but 🤷‍♀

kangabell commented on sass/elements/_elements.scss:
```diff
@@ -40,65 +40,67 @@ figure {
-	}
+  display: none;
+  visibility: hidden;
+	//@include full-width;
```
what? why are we hiding this?

jhogue replied:
Because clients are gonna client

kangabell replied:
hm  okay... Can you please remove from the DOM. And then just remove this entire block of text. I don't leave commented-out code in my repo. It should just be removed.

jhogue replied:
Took a different approach. See latest push.

jhogue committed:
Variablize

kangabell commented on sass/typography/_typography.scss:
```diff
@@ -2,7 +2,7 @@ html {
 	-webkit-font-smoothing: antialiased;
 	font-smoothing: antialiased;
-	font-size: 16px;
+	font-size: 100%;
```
disagree. tested earlier version (`16px`) and it does not override the browser setting (Firefox & Chrome). are you actually seeing this issue?

kangabell replied:
nevermind, was testing in an environment that was overriding the 16px oops. it's fine.

jhogue committed:
Better approach: Leave it but allow empty

Allow the widget area to be empty by moving the interior padding to a child element. If it is empty, only the border will show. If it is not, interior padding on the widget will be added back in place to achieve the design we had previously.

kangabell committed:
fix indent inconsistency
